pages/common/tee.md

```diff
@@ -17,4 +17,4 @@
 
 - Create a directory called "example", count the number of characters in "example" and write "example" to the terminal:
 
-`echo "example" | tee >(xargs mkdir) >(wc -c)`
+`dir_name="{{example}}"; echo "${dir_name}"; mkdir -- "${dir_name}"; echo "${#dir_name}"`
```

Comment:
This command no longer contains `tee`, which makes it obsolete for `tee`'s tldr page

Reply:
Deletion then? I see little to no use to the example.

Reply:
I think that the example is useful, because it demonstrates how tee can be used in conjunction with process substitution in order to achieve powerful behaviour. As mentioned above, I do not think that we should eliminate process substitution from our examples.

Reply:
As a beginner, I don't think that I would ever need to run this example but I think we should keep it because it may be useful for learning.

Reply:
I added a portability note